tools/devtool

```diff
@@ -500,6 +509,54 @@ cmd_build() {
     return $ret
 }
 
+cmd_strip() {
+    profile="release"
+    libc="musl"
+    target="$(uname -m)-unknown-linux-${libc}"
+
+    # Parse any command line args.
+    while [ $# -gt 0 ]; do
+        case "$1" in
+            "-h"|"--help")  { cmd_help; exit 1;     } ;;
+            "--target-libc")
+                shift
+                [[ "$1" =~ ^(musl|gnu)$ ]] || \
+                    die "Invalid libc: $1. Valid options are \"musl\" and \"gnu\"."
+                libc="$1"
+                target="$(uname -m)-unknown-linux-${libc}"
+                ;;
+            *)
+                die "Unknown argument: $1. Please use --help for help."
+            ;;
+        esac
+        shift
+    done
+
+    # Check prerequisites
```

Comment:
Another prerequisite would be to check that release binaries exist at the expected location, right?

Reply:
I've added a check for release binaries existance.